packages/core/docs/advanced/extending-graphql-queries.md

```diff
@@ -1,4 +1,4 @@
-# Extending GraphQL Queries
+# Extending GraphQL queries
```

Comment:
Actually, that might not necessarilly be a bad thing to write titiles using capital letters. In many places, e.g. newspaper articles, etc. or even documentation sites, titles/headers are being capitalized. Still seems more like a choice than only good way to do it.